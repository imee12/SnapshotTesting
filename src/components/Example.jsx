import React from 'react';


const Example = () => (
  <div className="dropdown--page">
    <h1>EXAMPLESSSS</h1>

  </div>
);

export default Example;
