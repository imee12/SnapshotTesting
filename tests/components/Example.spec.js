import React from 'react';
import renderer from 'react-test-renderer';
// import { shallow, mount } from 'enzyme';
// import toJSON from 'enzyme-to-json';
import Example from '../../src/components/Example';



describe('Example component', () => {
  test('renders a div', () => {
    const component = renderer.create(
      <Example>EXAMPLE</Example>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
