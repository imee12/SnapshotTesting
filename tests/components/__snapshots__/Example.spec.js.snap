// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button component renders a div 1`] = `
<div
  className="dropdown--page"
>
  <h1>
    EXAMPLESSSS
  </h1>
</div>
`;

exports[`Example component renders a div 1`] = `
<div
  className="dropdown--page"
>
  <h1>
    EXAMPLESSSS
  </h1>
</div>
`;
